velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -131,8 +131,15 @@ std::shared_ptr<WriterProperties> getArrowParquetWriterOptions(
   if (!options.enableDictionary) {
     properties = properties->disable_dictionary();
   }
-  properties =
-      properties->compression(getArrowParquetCompression(options.compression));
+  if (options.columnCompressionsMap.empty()) {
```

Comment:
I don't think it hurts to have the default compression.
Users could specify only a few columns.

Reply:
Can you verify what happens if only a few columns are specified?
Probably add 2 extra columns to the current test.

Reply:
So I verified and it does not hurt to have the default compression. 
I added the 2 extra columns to the current test and both of them have a `compressionKind` of `common::CompressionKind_NONE` which is as desired.

I also specified both the `compression`  and the `columnCompressionsMap` writer options at the same time (only a few columns) and and the columns for which a compression wasn't specified in the `columnCompressionsMap`  matched the `compression` specified in the `compression writerOptions`. 

In short, it works.

Reply:
> I added the 2 extra columns to the current test and both of them have a compressionKind of common::CompressionKind_NONE `

Let's add a non-default to the current test for compression.